sw/device/silicon_creator/rom_ext/doc/ownership.md

```diff
@@ -40,13 +40,60 @@ A chip may be in one of the following states:
+    direction LR
+    start[LockedOwner] --> unlock{BootSvc::OwnershipUnlock}
+    unlock -->|mode: Any| any[UnlockedAny]
+    unlock -->|mode: Endorsed| endorsed[UnlockedEndorsed]
+    unlock -->|mode: Update| self[UnlockedSelf]
+    any --> activate{BootSvc::OwnershipActivate}
+    endorsed --> activate
+    self --> activate
+    activate --> done[LockedOwner]
+    style container fill:none,stroke:black
+end
+```
+
+### Owner Configuration
+
+The owner configuration (sometimes called "owner block") allows an owner to tailor the configuration of the chip to their use case.
+The configuration allows the owner to control the flash configuration, application verification keys, rescue mode configuration and miscellaneous configuration of the chip.
+
+- The flash configuration allows the owner to set access control (read, program, erase) and storage policy (ECC, scrambling, high endurance) of various regions and INFO pages of flash.
+- The owner may add a number of application verification keys that permit code execution.
+  The application key configuration includes an execution domain (e.g. `prod`, `dev` or `test`) and a key diversifier constant allowing the owner to control the derivation of sealing keys by the OpenTitan key manager.
+- The rescue configuration allows the owner to configure the opertion of the rescue mode.
+  The configuration allows protecting flash regions from erasure during rescue and designating the allowed set of rescue commands (an owner may want to disable certain rescue commands if they feel those commands represent an attack or denial of serivce vector in their use case).
+- The miscellaneous configuration contains configuration for features that aren't well categorized into one of the above configurations:
+  - Enable or disable SRAM execution.
+  - Reset the BL0 minimum security version.
+
+#### Update Mode
+
+There are several allowed update modes for the ownership configuration.
+These update modes are motivated by two scenarios:
+
+- The Unlock Key is a very powerful key and owners may want to attenuate the power of the unlock key to mitigate the risk of the key leaking.
+- An owner may want to provide an update mechanism for devices in the field that does not require the unlock and activate procedure.
+
+The update mode provides for three types of owner configuration update procedures:
+
+- `Open` mode allows the unlock key to unlock the device into any of the unlocked states.
+- `Self` mode allows the unlock key to unlock the device into only the `UnlockedSelf` state.
+- `NewVersion` mode forbids the unlock key from unlocking the device.
+  Instead, the device may accept an ownership configuration update at any time _if and only if_ the new configuration is signed by the self-same owner _and_ the configuration has a newer `config_version` number.
+
+When the update mode is either `Self` or `NewVersion`, the power of the unlock key is reduced to only allow ownership operations that do not change the owner.
```

Comment:
Maybe worth an extra sentence here about how the power of the unlock key is reduced? My understanding is that in  `Self` or `NewVersion` the attacker would need to steal the owner key to take over ownership of the device, which contains the implicit assumption that the owner key is stored more securely than the unlock key. I think it's worth being explicit about this so that people know they need to protect the owner key in order to get the benefits of update mode.

Reply:
I added a section describing the owner key and included the example of storing the key in an offline HSM.

Can you clarify what you're asking for with "how the power of the unlock key is reduced"?  I think the text already describes the reduction in power of the unlock key:
- When in `Self` mode, the unlock key can only unlock to the `UnlockedSelf` state, meaning you can only update the config, not perform a full ownership transfer.
- When in `NewVersion` mode, you cannot move to an unlocked state at all.

Reply:
I think I made a poor choice of words here -- I meant something like "how reducing the power of the unlock key increases security". What I was getting at is that if the owner key is no safer than the unlock key then you don't really win much, and the new section resolves this by making the expectation that the owner key is safer than the unlock key explicit. Thanks!